# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pwlib/pldap.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
===================================================================
RCS file: /cvsroot/openh323/pwlib/src/ptclib/pldap.cxx,v
retrieving revision 1.17
retrieving revision 1.18
diff -u -r1.17 -r1.18
--- pwlib/src/ptclib/pldap.cxx	2004/05/24 12:02:49	1.17
+++ pwlib/src/ptclib/pldap.cxx	2005/08/31 15:21:34	1.18
@@ -24,6 +24,9 @@
  * Contributor(s): ______________________________________.
  *
  * $Log: pldap.cxx,v $
+ * Revision 1.18  2005/08/31 15:21:34  dominance
+ * fix building with recent OpenLDAP
+ *
  * Revision 1.17  2004/05/24 12:02:49  csoutheren
  * Add function to permit setting a limit on the number of results returned
  * from an LDAP query. Change the default number of results to unlimited,
@@ -89,6 +92,7 @@
 
 #if P_LDAP
 
+#define LDAP_DEPRECATED 1
 #include <ldap.h>
 
 
